test-commons/src/main/scala/za/co/absa/spline/scalatest/ConditionalIgnore.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.scalatest
+
+import org.scalatest.{Ignore, Tag}
+import org.apache.spark.SPARK_VERSION
+
+object ConditionalTestIgnore {
+
+  private def ignoreTestBySparkVersion(condition: String => Boolean): Tag = {
+    val matches = condition(SPARK_VERSION)
+    Tag(if (matches) "" else classOf[Ignore].getName)
+  }
+
+  //Ignores unit test when version of spark is 2.2 or below.
```

Comment:
This comment is both misleading (should be exactly 2.2) and redundant. The line of code is a self-explanatory.